build_defs/go.build_defs

```diff
@@ -1242,14 +1242,15 @@ def go_repo(module: str, version:str='', download:str=None, name:str=None, insta
     requirements = " ".join(requirements)
 
     build_tag_args = '--build_tag ' + ' --build_tag '.join(build_tags) if build_tags else ''
+    label_args = '--label ' + ' --label '.join(labels) if labels else ''
```

Comment:
```suggestion
    label_args = [f"--label {label}" for label in labels].join(" ")
```

Reply:
I have to think really hard about the precedence to the ternary operation to be convinced that this works as intended; I think this is more clear. I would change the line above to match as well.

Reply:
Sure, makes sense.